package.json

```diff
@@ -9,35 +9,38 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "type-check": "tsc",
+    "typecheck": "tsc --noEmit",
     "lint": "eslint . --ext .ts,.tsx",
     "lint:fix": "eslint . --ext .ts,.tsx --fix"
   },
   "dependencies": {
-    "@prezly/sdk": "^2.15.0",
-    "@prezly/slate-renderer": "^0.1.2",
-    "next": "^11.1.2",
-    "next-seo": "^4.28.0",
+    "@hcaptcha/react-hcaptcha": "^1.1.0",
+    "@prezly/content-renderer-react-js": "0.10.2",
+    "@prezly/sdk": "^6.14.0",
+    "@prezly/slate-types": "^0.14.6",
+    "@prezly/theme-kit-nextjs": "1.3.2",
+    "@prezly/themes-intl-messages": "^1.8.2",
+    "@prezly/uploadcare-image": "^0.3.1",
+    "algoliasearch": "^4.12.1",
+    "next": "^12.0.10",
+    "next-seo": "^5.1.0",
     "parse-data-url": "^4.0.1",
     "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "react-dom": "^17.0.1",
+    "react-instantsearch-dom": "^6.22.0",
+    "react-intl": "^5.24.6",
+    "react-use": "^17.3.2",
+    "string-strip-html": "^9.1.6"
   },
   "devDependencies": {
+    "@prezly/eslint-config": "4.0.0",
     "@types/node": "^16.10.2",
     "@types/parse-data-url": "^3.0.0",
     "@types/react": "^17.0.4",
     "@types/react-dom": "^17.0.1",
-    "@typescript-eslint/eslint-plugin": "^4.22.0",
-    "@typescript-eslint/parser": "^4.22.0",
-    "eslint": "^7.25.0",
-    "eslint-config-airbnb-typescript": "^12.3.1",
-    "eslint-import-resolver-typescript": "^2.4.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-react": "^7.23.2",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "next-logger": "^2.2.0",
-    "typescript": "^4.4.3"
+    "@types/react-instantsearch-dom": "^6.12.2",
+    "next-logger": "^3.0.0",
+    "typescript": "^4.5.5"
```

Comment:
`prettier` dependency is missing here. Also worth adding scripts for prettier checks

Reply:
Added 👍